pkg/install/disableupdates_test.go

```diff
@@ -0,0 +1,58 @@
+package install
+
+import (
+	"context"
+	"testing"
+
+	v1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/client-go/config/clientset/versioned/fake"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestDisableUpdates(t *testing.T) {
+	ctx := context.Background()
+
+	versionName := "version"
+
+	configcli := fake.NewSimpleClientset()
+	configcli.ConfigV1().ClusterVersions().Create(
+		&v1.ClusterVersion{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: versionName,
+			},
+		},
+	)
+
+	cv := getClusterVersion(t, configcli, versionName)
+	cv.Spec.Upstream = "RemoveMe"
+	cv.Spec.Channel = "RemoveMe"
+
+	_, err := configcli.ConfigV1().ClusterVersions().Update(cv)
```

Comment:
Is it possible to set `Upstream` and `Channel` values in the `Create` call and remove the `Update` call?

Reply:
Or maybe we can even get rid of the `Create` call and do something like:

```go
configcli := fake.NewSimpleClientset(&v1.ClusterVersion{
    # ...
})
```

Reply:
I will try it out, thanks.

Reply:
@m1kola Changed. I took it a step further and just put everything inside the `i := &Installer` assignment.